src/mlpack/methods/ann/layer/concatenate.hpp

```diff
@@ -14,44 +14,47 @@
 #define MLPACK_METHODS_ANN_LAYER_CONCATENATE_HPP
 
 #include <mlpack/prereqs.hpp>
-#include <mlpack/methods/ann/layer/layer_traits.hpp>
+#include "layer.hpp"
 
 namespace mlpack {
 namespace ann /** Artificial Neural Network. */ {
 
 /**
  * Implementation of the Concatenate module class. The Concatenate module
  * concatenates a constant given matrix to the incoming data.
- * Note: Users need to use the Concat() function to provide the concat matrix.
  *
- * @tparam InputDataType Type of the input data (arma::colvec, arma::mat,
- *         arma::sp_mat or arma::cube).
- * @tparam OutputDataType Type of the output data (arma::colvec, arma::mat,
- *         arma::sp_mat or arma::cube).
+ * The Concat() function to provide the concat matrix, or it can be passed to
```

Comment:
```suggestion
 * The Concat() function provides the concat matrix, or it can be passed to
```

Not sure it makes sense now, but it sounded confusing to me.